app/models/ability.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    can :read, :all
+
+    if user.present?
+      can [:new, :create, :read, :edit, :update, :destroy], CharacterType, user_id: user.id
```

Comment:
@StefanGajic see if you can replace all of these permissions with the `manage` keyword.